app/models/space.rb

```diff
@@ -86,7 +86,7 @@ def user_info
   end
 
   def can_create_private_models
-    if is_private && !user.try(:can_create_private_models)
+    unless is_public? || organization_id || user.try(:can_create_private_models)
```

Comment:
no worries now, but the ruby way is more to have a method for ``belongs_to_organization?``